ansible_events_ui/db/migrations/script.py.mako

```diff
@@ -17,8 +18,15 @@ depends_on = ${repr(depends_on)}
 
 
 def upgrade() -> None:
-    ${upgrades if upgrades else "pass"}
-
+    schema_upgrades()
+    if context.get_x_argument(as_dictionary=True).get('create_default_user', None):
```

Comment:
`create_default_user` is a local option. It definitely shouldn't go into a template.